src/main/java/com/minecolonies/util/SchematicWrapper.java

```diff
@@ -548,7 +548,7 @@ public boolean findNextBlockToClear()
 
         }
         //Check for air blocks and if blocks below the hut are different from the schematicWorld
-        while((worldBlockAir() || (progressPos.getY() <= getOffset().getY() && doesSchematicBlockEqualWorldBlock())) && count < Configurations.maxBlocksCheckedByBuilder);
+        while((worldBlockAir() ||  doesSchematicBlockEqualWorldBlock()) && count < Configurations.maxBlocksCheckedByBuilder);
```

Comment:
We don't want to do this, we kept this so that we don't create giant holes in the ground. In theory the placeholder block should fix this, but we haven't implemented it fully in all our schematics.